Mathlib/Util/Qq.lean

```diff
@@ -40,4 +40,18 @@ def findLocalDeclWithTypeQ? {u : Level} (sort : Q(Sort u)) : MetaM (Option Q($so
 This is a Qq version of `Lean.Meta.mkDecideProof`. -/
 def mkDecideProofQ (p : Q(Prop)) : MetaM Q($p) := mkDecideProof p
 
+/-- Join a list of elements of type `α` into a container `m α`.
+
+Usually `m` is `q(Multiset)` or `q(Finset)` or `q(Set)`. -/
+def mkSetLiteralQ {u v : Level} {α : Q(Type u)} (m : Q(Type u → Type v))
+    (elems : List Q($α))
+    (_ : Q(EmptyCollection ($m $α)) := by exact q(inferInstance))
+    (_ : Q(Singleton $α ($m $α)) := by exact q(inferInstance))
+    (_ : Q(Insert $α ($m $α)) := by exact q(inferInstance)) :
+    Q($m $α) :=
```

Comment:
Can't you take in `m a` directly?
```suggestion
def mkSetLiteralQ {u v : Level} {α : Q(Type u)} (β : Q(Type v))
    (elems : List Q($α))
    (_ : Q(EmptyCollection $β) := by exact q(inferInstance))
    (_ : Q(Singleton $α $β) := by exact q(inferInstance))
    (_ : Q(Insert $α $β) := by exact q(inferInstance)) :
    Q($β) :=
```

Reply:
You seem to have rebased the change away 😅

Reply:
Fixed:)